elections/2023-SC/README.md

```diff
@@ -0,0 +1,163 @@
+
+## Schedule
+
+| Date         | Event                    |
+| ------------ | ------------------------ |
+| October 17   | Announcement of Election, call for nominations, exceptions |
+| November 1   | All candidate nominations due by 23:59 UTC (3:59pm Pacific) |
+| November 3   | Election Begins via Elekto UI |
+| November 28  | Voter exception requests due by 23:59 UTC (3:59pm Pacific) |
+| November 30  | Election Closes by 23:59 UTC (3:59pm Pacific) |
+| December 6   | Announcement of Results |
+
+## Election Officers
+
+In an effort to run a fair and transparent election, the following people
+have been asked by the Steering Committee to run this election:
+
+- [Dawn Foster](https://github.com/geekygirldawn)
+- [Reto Lehmann](https://github.com/ReToCode)
+
+You can reach us by emailing elections@knative.team
+
+## Candidacy Process
+
+**Nomination**
+
+If you want to stand for election for a Contributor Seat, open a PR against the
+[knative/community repository](https://github.com/knative/community) to include
+your candidate profile in the `/elections/2023-SC` folder, with the following
+filename format:
+
+```
+candidate-githubid.md
+```
+
+This profile should include:
+
+* Your name
+* Your company affiliation (employer or otherwise)
+* Your contributions to Knative
+* Why you are running
+
+You can find [a sample template in the folder](./nomination-template.md).
+
+Once you have created the PR, you may email knative-dev and/or knative-users
+*once* to let people know about your candidacy and encourage endorsements as
+comments on the PR. Please use something like this for your email in order to
+encourage endorsements in Github and not on the mailing list:
+
+```
+Fellow Knative community members,
+
+My name is {your name} and I am running for Steering Committee.  You can read
+my profile here: {link to PR}.  If you support my candidacy, please endorse me
+by commenting on that PR.  Please do not reply to endorse me on this list.
+```
+
+After a candidate has met all election requirements, the Election Officers will
+merge the profile PR.
+
+If you want to nominate someone else, you may do so, but PLEASE talk to them
+first.
+
+**Campaigning**
+
+Please refer to the [SC Election Charter] and understand
+that we care deeply about [limiting corporate campaigning]. The election
+officers and members of the steering committee [pledge to recuse] themselves
+from any form of electioneering.
+
+You should be running as a "brand free" individual, based on your contribution
+to the project as a member of this community, outside of whatever corporate
+roles you may hold.
+
+## Voting Process
+
+Elections will be held using [Elekto], an online voting tool created by CNCF 
+intern Manish Sahani.  Elekto has one critical advantage over CIVS, which is
+that it relies on GitHub Oauth to log you in to vote, instead of relying on 
+email. More details on voting are in the [Elekto documentation].
```

Comment:
```suggestion
Elections will be held using [Elekto], an online voting tool created
as a CNCF internal project. As a reminder it relies on GitHub Oauth to log you
in to vote, instead of relying on email. More details on voting are in the [Elekto documentation].
```